lib/pointing_party_web/live/card_live.ex

```diff
@@ -0,0 +1,114 @@
+defmodule PointingPartyWeb.CardLive do
+  use Phoenix.LiveView
+
+  alias PointingParty.{Card, VoteCalculator}
+  alias PointingPartyWeb.{Endpoint, Presence}
+
+  def render(assigns) do
+    Phoenix.View.render(PointingPartyWeb.CardView, "index.html", assigns)
+  end
+
+  def mount(%{username: username}, socket) do
+    Endpoint.subscribe("users")
```

Comment:
Can we set the topic as a module attribute and then re-use that when we need to broadcast and subscribe? `@topic "pointing_party"` for example.